### ISSUE TITLE
[REQ] [Protobuf-Schema] Protobuf available inside OpenAPI Generator Online

### ISSUE THREAD

**Issue by dobl1** [OpenAPITools/openapi-generator#7243]
Hi,

How to proceed to add existing converter module to OpenAPI Generator Online ? As protobuf-schema for example.

**Comment by wing328** [comment]
Have you tried the "latest master" of the online generator?

**Comment by dobl1** [comment]
Yes, I noticed nothing is returned trying to get details about generation options

![image](https://user-images.githubusercontent.com/9740002/90600641-39bbc480-e1f7-11ea-99e0-244ec6ac3866.png)

**Comment by wing328** [comment]
@dobl1 I just checked the source code of openapi-generator-online and confirmed that you're right - there's no endpoints for schema/doc generator.

Please use the CLI for the time being.

**Comment by wing328** [comment]
@dobl1 if you've time to contribute a fix, I can show you a couple of good starting points.
